ddl/backfilling_scheduler.go

```diff
@@ -66,11 +101,29 @@ func newBackfillScheduler(ctx context.Context, info *reorgInfo, sessPool *sessio
 		workers:      make([]*backfillWorker, 0, variable.GetDDLReorgWorkerCounter()),
 		taskCh:       make(chan *reorgBackfillTask, backfillTaskChanSize),
 		resultCh:     make(chan *backfillResult, backfillTaskChanSize),
+	}, nil
+}
+
+func (b *txnBackfillScheduler) setupWorkers() error {
+	return b.adjustWorkerSize()
+}
+
+func (b *txnBackfillScheduler) sendTask(task *reorgBackfillTask) {
+	b.taskCh <- task
+}
+
+func (b *txnBackfillScheduler) drainTasks() {
```

Comment:
if use a select instead only for loop is better?
func (b *ingestBackfillScheduler) drainTasks() {
    for {
        select {
        case <-b.taskCh:
            // Discard message from channel
        default:
            // Channel is empty, exit loop
            return
        }
    }
}